RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -23,8 +23,8 @@
         )
     )
 
-from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
-run2_ECAL_2017.toReplaceWith(particle_flow_zero_suppression_ECAL, _particle_flow_zero_suppression_ECAL_2017)
+#from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
+#run2_ECAL_2017.toReplaceWith(particle_flow_zero_suppression_ECAL, _particle_flow_zero_suppression_ECAL_2017)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant